scripts/core/device.yml

```diff
@@ -128,7 +128,9 @@ members:
       desc: "[out] vendor id from PCI configuration"
     - type: uint32_t
       name: deviceId
-      desc: "[out] device id from PCI configuration"
+      desc: |
+            [out] device id from PCI configuration
+            Note, the least-significant 16-bits contain the device id.
```

Comment:
@wdamon-intel : I think we could be more generic and say this uses little-endian format.